### PR TITLE
Update helm to version 2.0.0-alpha.5

### PR DESCRIPTION
**Pull request by tback**
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/helm.rb [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-  url "https://github.com/kubernetes/helm/releases/download/v#{version}/helm-v#{version}-darwin-amd64.tar.gz"
+  # storage.googleapis.com/kubernetes-helm/helm-v was verified as official when first introduced to the cask (2016-10-11)
+  url "http://storage.googleapis.com/kubernetes-helm/helm-v#{version}-darwin-amd64.tar.gz"
   appcast 'https://github.com/kubernetes/helm/releases.atom',
-          checkpoint: 'bc0b7ead6b44231cb9312dbec9a147b3c202387cdfd828cee0546f1b986d623c'
+          checkpoint: '89bc384e94e30b530e8afce2db165db31c3e18465077de889255d166a9110d44'
   name 'Helm'
   homepage 'https://github.com/kubernetes/helm'
 
+  license :apache
```
Comment (vitorgalvao) [inline Casks/helm.rb]: Remove `license`, it has been deprecated and is no longer in use.

File: Casks/helm.rb [hunk 2]
```diff
@@ -1,12 +1,15 @@
 cask 'helm' do
-  version '2.0.0-alpha.4'
-  sha256 'fcba539993b7b1459a5c2e02c93e87c7f2fdb5fea941b4a0bc3367e58f85c47b'
+  version '2.0.0-alpha.5'
+  sha256 '6bd540611042665c78923fd722e9cd25d08e7381e982535a3a720458da163811'
 
-  url "https://github.com/kubernetes/helm/releases/download/v#{version}/helm-v#{version}-darwin-amd64.tar.gz"
+  # storage.googleapis.com/kubernetes-helm/helm-v was verified as official when first introduced to the cask (2016-10-11)
```
Comment (reitermarkus) [inline Casks/helm.rb]: Remove the date here.

Comment (vitorgalvao) [inline Casks/helm.rb]: Remove `helm-v` at the end.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by reitermarkus — CHANGES_REQUESTED** [review]

File: Casks/helm.rb (hunk 2 above)
Comment (tback) [inline Casks/helm.rb]: Removed

File: Casks/helm.rb (hunk 1 above)
Comment (tback) [inline Casks/helm.rb]: Removed

File: Casks/helm.rb (hunk 2 above)
Comment (tback) [inline Casks/helm.rb]: Removed

**Comment by tback** [comment]
Should be clear now.

**Pull request merged**